src/Components/CreateImageWizard/formComponents/Repositories.js

```diff
@@ -519,10 +530,10 @@ const Empty = ({ isFetching, refetch }) => {
         href={isBeta() ? '/preview/settings/content' : '/settings/content'}
         className="pf-u-mr-sm"
       >
-        Repositories
+        Go to repositories
       </Button>
       <Button
-        variant="primary"
+        variant="secondary"
```

Comment:
Sorry if it's a dumb question, but why the variant had to change ?

Reply:
To bring the step closer to the mocks below. I think the idea is to differentiate between the buttons a bit and also [design guidelines](https://www.patternfly.org/components/button/design-guidelines) seem to not put two primary buttons right next to each other as well.

![image](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/154aef1f-a5de-4f34-90c2-fa301de2bc2d)